[Expo] Failed to resolve plugin for module "react-native-email-link"

Expo is unable to resolve the `react-native-email-link` plugin. 

Steps to reproduce:
1 - `expo init`
2 - Add `"plugins": ["react-native-email-link"]` to app config.
3 - `expo config --type prebuild`


Note: `expo --version: 4.12.1`

I am getting a similar issue with using babel.config.js file for plugins. The error says `Cannot find module 'babel-plugin-react-native-email-link'` 

```
module.exports = function (api) {
  api.cache(true);
  return {
    presets: ["babel-preset-expo"],
    plugins: [
      "react-native-email-link",
    ],
  };
};
```

I was getting this error also. ESLint was also failing for us with this error:
`
Error: Cannot find module '/Users/ryanhyslop/Developer/my-sports-wellbeing-app/node_modules/react-native-email-link/index'. Please verify that the package.json has a valid "main" entry
`

Looking into the `package.json` we have `main: index` set but there is no `index.js` in the project, just `index.(ios|android).js`

Now this could be effecting us due to some local configuration/environment issues and I'm not sure what the best course of action here is but adding an empty `index.js` file into the repo resolves both issues for us? Perhaps the index could conditionally require the platform index, even if it is a bit redundant? 🤷

--- a/index.js
+++ b/index.js
@@ -1,0 +1,1 @@
+// Placeholder file so that Expo plugin module resolution works

